README.md

```diff
@@ -35,8 +35,6 @@
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> The core concepts of HTML,CSS and git hub are implemented.These three tools are essential for modern web development and play a critical role in building high-quality websites and applications.
-
 ### Key Features <a name="key-features"></a>
```

Comment:
- [ ] I asked for changes and even gave a suggestion on how to do so, but it seems you forgot to add the technologies back. 
-  [ ] As I said earlier, follow this format to make it look professional. just edit the sites inside I've done for that of the `html` to show you how, you can add that of `CSS` and any other technology used 

e.g
```<summary>Client</summary>
  <ul>
    <li><a href="https://W3Schools.org/">HTML</a></li>
  </ul>
</details>
```